### PR TITLE
Interface support

### PR DESCRIPTION
**Pull request by kubijo**
Hi.

This is an unfinished pull request addressing #173 
Added unit test seems to work fine (I've just delegated `InterfaceDeclaration` to `ObjectTypeAnnotation`.
What I haven't been able to update yet is creating the export...

### CODE REVIEW

File: src/convertToPropTypes.js [hunk 1]
```diff
@@ -89,15 +89,18 @@ export default function convertToPropTypes(node, importedTypes, internalTypes) {
...
   else if (node.right) {
     node = node.right;
   }
 
-  if (node.type === 'ObjectTypeAnnotation') {
+  if (node.type === 'InterfaceDeclaration') {
+    node = node.body;
+  }
+
+  if (node.type === 'ObjectTypeAnnotation' || node.type === 'ObjectTypeAnnotation') {
```
Comment (brigand) [inline src/convertToPropTypes.js]: Typo?

**Comment by brigand** [comment]
Just a few minutes ago I extracted the code to make export statements into a function `addExportTypeDecl`. Usage: `addExportTypeDecl(path, 'bpfrpt_proptype_Something')`. You create the name with `getExportNameForType` like `getExportNameForType('Something')`, which you also use to create the local variable which will be exported.

File: src/convertToPropTypes.js (hunk 1 above)
Comment (kubijo) [reply src/convertToPropTypes.js]: yep, will fix

**Comment by kubijo** [comment]
I see.
I've rebased my changes and now I'd like to finish this up with that export stuff.

**Comment by kubijo** [comment]
Sorry, for the force pushes ... now it seems to work fine even when I test it on my project.
However now there are some failing tests ...

**Pull request merged**